packages/backend/src/services/GitIntegrationService/GitIntegrationService.ts

```diff
@@ -440,91 +462,92 @@ Affected charts:
         });
     }
 
-    async createPullRequestForCustomMetrics(
+    private async getGithubProps(
```

Comment:
extracted method from createPullRequestFromSql  to be reused in createPullRequestForCustomMetrics